apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx

```diff
@@ -120,7 +116,7 @@ export default class LibraryManagerDialog extends React.Component {
         <LibraryListItem
           key={library.name}
           library={library}
-          onRefresh={this.refreshLibrary}
+          onRefresh={undefined}
```

Comment:
nit: cleaner to just remove this line?

Reply:
I'd prefer to leave it here - helps me remember where it needs to be called for when I implement onRefresh.